qiskit/transpiler/passes/optimization/echo_rzx_weyl_decomposition.py

```diff
@@ -10,7 +10,7 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Weyl decomposition of two-qubit gates in terms of echoed cross-resonance gates."""
+"""Weyl-decomposition of two-qubit gates in terms of echoed cross-resonance gates."""
```

Comment:
```suggestion
"""Weyl decomposition of two-qubit gates in terms of echoed cross-resonance gates."""
```

The hyphen shouldn't be here, because "Weyl decomposition" is two words, and isn't being used as a compound adjective.  That said, this is one of the docstrings that isn't built into the docs, so it doesn't really matter.